README.md

```diff
@@ -73,6 +73,15 @@ Install the extension according to your browser:
 - use <kbd>tab</kbd> and <kbd>Shift+tab</kbd> to navigate between the shortcodes and emojis
 - press <kbd>Enter</kbd> to copy
 
+## 🧩 Features
+
+- 🔍 Search for a specific gitmoji's code/emoji by name or description
+- 🌔 Dark/light mode
+- ❓ Show gitmoji's description on mouse over or focus
+- ❌ Close extension's popup after a gitmoji is saved in the clipboard
```

Comment:
This one is redundant I think

Reply:
I wanted to specify it because you can disable this behaviour in the settings page. So is there à reason to not keep it?

Reply:
Done it.